### PR TITLE
Add `mutate` `--ports` option to set the exposed ports

### PR DESCRIPTION
**Pull request by josephglanville**
Allows configuring of `ExposedPorts` via `mutate`.

### CODE REVIEW

File: cmd/crane/cmd/mutate.go [hunk 1]
```diff
@@ -172,6 +182,7 @@ func NewCmdMutate(options *[]crane.Option) *cobra.Command {
 	mutateCmd.Flags().StringSliceVar(&newLayers, "append", []string{}, "Path to tarball to append to image")
 	mutateCmd.Flags().StringVarP(&user, "user", "u", "", "New user to set")
 	mutateCmd.Flags().StringVarP(&workdir, "workdir", "w", "", "New working dir to set")
+	mutateCmd.Flags().StringSliceVar(&ports, "ports", nil, "New ports to set")
```
Comment (jonjohnsonjr) [inline cmd/crane/cmd/mutate.go:185]:
```suggestion
	mutateCmd.Flags().StringSliceVar(&ports, "exposed-ports", nil, "New ports to expose")
```

**Comment by jonjohnsonjr** [comment]
Need to run `./hack/update-codegen.sh` again

**Comment by jonjohnsonjr** [comment]
Looks like https://github.com/google/go-containerregistry/pull/1683 has merge conflicts in the generated docs, need a rebase (sorry 😅).

**Comment by josephglanville** [comment]
@jonjohnsonjr rebased.

**Review by jonjohnsonjr — APPROVED** [review]

**Pull request merged**